src/etc/inc/util.inc

```diff
@@ -2536,4 +2536,39 @@ function validateipaddr(&$addr, $type, $label, &$err_msg, $alias=false) {
 
 	return false;
 }
+* returns true if $dhcp6duid is a valid duid entrry */
```

Comment:
It's missing / before * to become a comment. Also please leave a blank line between function definitions